whiteblock/cmd/transactions.go

```diff
@@ -17,6 +17,7 @@ var (
 	toFlag       string
 	fromFlag     string
 	nodeFlag     string
+	dataFlag	 string
```

Comment:
Don't add to the global namespace, I am working to remove all the flags from the namespace. If you would like to see how to use the flags without namespace polution, [here is a lot of examples](https://github.com/whiteblock/cli/blob/dev/whiteblock/cmd/build/flags.go).